### DISCUSSION TITLE
what is it mean by the arrow at the git log in intellij IDEA

### DISCUSSION THREAD

**Discussion by developerSeonah** [community/community#143267] [Codespaces]
![image](https://github.com/user-attachments/assets/3b20a1bb-896b-43b5-b58b-d12d9c78f527)

at the git log

just found out the arrow that just appears 

what it is mean by then

**Comment by thiha-min-thant** [comment]
Long Edges: if this option is enabled, long branches are displayed in full, even if there are no commits in them. If this option is disabled (by default), long branches are replaced with a down arrow.

https://www.jetbrains.com/help/idea/log-tab.html#CommitsPane

**Reply by developerSeonah** [reply]
thanks!
